x-pack/plugins/lens/public/trigger_actions/open_lens_config/in_app_embeddable_edit/in_app_embeddable_edit_action_helpers.tsx

```diff
@@ -104,52 +105,55 @@ export async function executeEditEmbeddableAction({
     };
     onUpdate(newAttributes);
   };
+  core.theme.theme$.subscribe(async (theme) => {
```

Comment:
Probably want to avoid `async` here - if the `getEditLensConfiguration` call fails, there's no error handling (subscribe doesn't care what you return from the function)